library/src/main/java/org/apache/apex/malhar/lib/window/Tuple.java

```diff
@@ -45,11 +45,12 @@
    */
   class PlainTuple<T> implements Tuple<T>
   {
-    private T value;
```

Comment:
It depends on how PlainTuple is used later. If it is used as a key, making PlainTuple immutable may be helpful. But I guess, T itself may be used as a key, instead of PlainTuple. In cases mutable or immutable use case becomes common, we may introduce (Im)MutablePlainTuple.